integration-web/src/test/java/org/symphonyoss/integration/web/filter/WebHookOriginCheckFilterTest.java

```diff
@@ -60,6 +61,12 @@
 
   private static final String REMOTE_ADDRESS = "192.30.252.40";
 
+  private static final String REMOTE_ADDRESS_LIST_FIRST_IP_ALLOWED = "192.30.252.40, 168.140.252.55";
+
+  private static final String REMOTE_ADDRESS_LIST_SECOND_IP_ALLOWED = "10.30.224.40, 192.30.252.40";
```

Comment:
One of these should verify that the regex will work for IPs that are separated by comma but no space.